invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -73,25 +72,36 @@ class CreatibutorsFieldForm extends Component {
     const creatibutorsError =
       error || (creatibutorsList === formikInitialValues && initialError);
 
+    let className = "";
```

Comment:
I remember we discussed this couple of times, from my understanding you can also do only
```suggestion
    const className = contributors.severity;
```
because you also pass errors in the new format